assets/prometheus-user-workload/prometheus.yaml

```diff
@@ -119,6 +119,8 @@ spec:
   overrideHonorLabels: true
   overrideHonorTimestamps: true
   podMetadata:
+    annotations:
+      workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
```

Comment:
Do we actually want the user-workload bits treated as a management workload here? Iiuc the user-workload bits are provided as best-effort, so is this maybe a reason to not mark them as management workloads?

Reply:
I think management is appropriate, even if they're best-effort.  We want them on the dedicated management cores and not taking CPU that should be instead guaranteed to user workloads.

Reply:
I agree with @lack: everything that is managed by CMO falls into the "management" bucket.